pkg/project/v2/project_loader.go

```diff
@@ -98,16 +104,18 @@ func LoadProjects(fs afero.Fs, context ProjectLoaderContext, specificProjectName
 		}
 		seenProjectNames[projectNameToLoad] = struct{}{}
 
-		projectDefinition, found := context.Manifest.Projects[projectNameToLoad]
+		projectDefinition, found := loaderContext.Manifest.Projects[projectNameToLoad]
 		if !found {
 			continue
 		}
 
-		project, errs := loadProject(workingDirFs, context, projectDefinition, environments)
+		project, errs := loadProject(ctx, workingDirFs, loaderContext, projectDefinition, environments)
```

Comment:
Idea to remove naming conflict:
```suggestion
		project, loadProjectErrs := loadProject(ctx, workingDirFs, loaderContext, projectDefinition, environments)
```